SampleApplications/Workshop/DataTypes/README.md

```diff
@@ -0,0 +1,7 @@
+﻿## DataTypes Server
+
+This server is build using the UA-.NETStandard stack as example, how to use data types.
```

Comment:
small adding: 
This server is build using the UA-.NETStandard stack as example of how to define and use custom data types.

Reply:
updated the text